search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeService.kt

```diff
@@ -428,7 +428,7 @@ class TemporalEntityAttributeService(
                         WHEN attribute_type = 'Property' AND attribute_value_type IN ('DATETIME', 'DATE', 'TIME') THEN 
                             jsonb_path_exists(temporal_entity_attribute.payload,
                                 '$."$NGSILD_PROPERTY_VALUE" ? 
-                                    (@."$JSONLD_VALUE_KW".datetime() ${query.second} $targetValue.datetime())')
+                                    (@."$JSONLD_VALUE_KW".datetime() ${query.second} $targetValue)')
```

Comment:
don't see what it has to do with the PR?

Reply:
When I have this, for exemple if targetValue = "Paris" the request does this `"Paris".datetime()"' but PostgreSQL don't like that. It returns errors at the level of `.`

Reply:
This is weird since it should only go there if attribute is of type `DATETIME`, `DATE` or `TIME`. There is another problem somewhere else...